csharp/random-writer/src/RandomWriter/RandomWriter.csproj

```diff
@@ -6,11 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Amazon.CDK" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.DynamoDb" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Events" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Events.Targets" Version="1.15.0-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.Lambda" Version="1.15.0-devpreview" />
+    <PackageReference Include="Amazon.CDK" Version="1.18.0-devpreview" />
```

Comment:
same as the other comment, there is no `1.18.0-devpreview` package

Reply:
Thanks!